### PR TITLE
[AL-6294] | Bug Fix | Support for setting a 'from cursor for dataset.data_rows()

### PR DESCRIPTION
**Pull request by vbrodsky**

### CODE REVIEW

File: labelbox/pagination.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ class _CursorPagination(_Pagination):
     def __init__(self, cursor_path: List[str], *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.cursor_path = cursor_path
-        self.next_cursor: Optional[Any] = None
+        self.next_cursor: Optional[Any] = kwargs.get('params', {}).get('from')
```
Comment (kopreschko) [inline labelbox/pagination.py:137]: This change makes the code a bit obscure. To enhance readability and maintainability, what do you think about explicitly adding params to the arguments of the constructor?

**Review by kopreschko — COMMENTED** [review]
Just added a small observation

File: labelbox/pagination.py (hunk 1 above)
Comment (vbrodsky) [reply labelbox/pagination.py:137]: I did add it in the past... but the parent init mypy had issues with it... ok let me look at it again

Comment (vbrodsky) [reply labelbox/pagination.py:137]: I had it like you suggested, but had issues similar to https://stackoverflow.com/questions/21764770/typeerror-got-multiple-values-for-argument with mypy.. so I would prob have to 're-add' it back into kwargs before calling the parent constructor. Let me see

**Review by kopreschko — APPROVED** [review]

File: labelbox/pagination.py [hunk 2]
```diff
@@ -131,10 +131,10 @@ def get_next_page(self) -> Tuple[Dict[str, Any], bool]:
 
 class _CursorPagination(_Pagination):
 
-    def __init__(self, cursor_path: List[str], *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, cursor_path: List[str], params={}, *args, **kwargs):
+        super().__init__(*args, **dict(kwargs, params=params))
```
Comment (kopreschko) [inline labelbox/pagination.py:135]: might be shorter to just use ```{**kwargs, **params}```
but I think this is getting weirder, it's probably better to leave it as it was before.

**Pull request merged**